scripts/generateAssetData/utils/portals.ts

```diff
@@ -131,8 +131,8 @@ export const getPortalTokens = async (nativeAsset: Asset): Promise<Asset[]> => {
     return {
       ...explorerData,
       color: colorMap[assetId] ?? '#FFFFFF',
-      icon: token.images?.[0] ?? '',
-      icons: token.images ?? [],
+      icon: token.images?.[0],
```

Comment:
Now that `icon` can be undefined, this keeps things saner - we were previously generating base64 assets, but now that we don't, we would generate an empty `icon` line in `generatedAssetData.json` for all assets without `icon`